pkg/console/operator/sync_v400.go

```diff
@@ -200,12 +200,13 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 func (co *consoleOperator) SyncConsoleConfig(consoleConfig *configv1.Console, consoleURL string) (*configv1.Console, error) {
 	oldURL := consoleConfig.Status.ConsoleURL
 	updated := consoleConfig.DeepCopy()
```

Comment:
We could move this inside the `if` since we only need to copy if we're doing an update